lib/rouge/lexers/racket.rb

```diff
@@ -511,7 +511,7 @@ def self.builtins
         rule %r/['`]#{id}/i, Str::Symbol
         rule %r/#\\([()\/'"._!\$%& ?=+-]{1}|[a-z0-9]+)/i,
           Str::Char
-        rule %r/#t(rue)?|#f(alse)?/, Name::Constant
+        rule %r/#t(rue)?|#f(alse)?/i, Name::Constant
```

Comment:
Note this would also represent #TRUE and #FALSE as valid booleans.

Reply:
That's right. I realise that might not be syntactically correct in Racket but it's a non-goal of Rouge to check syntax correctness. We will generally prefer simpler regular expressions that have no false-negatives (as it were) than more complicated expressions that are more difficult to debug and maintain.